### DISCUSSION TITLE
Not able to completely delete item with translations field

### DISCUSSION THREAD

**Discussion by kingleoric** [directus/directus#14431] [Archive]
### Describe the Bug

I use the translations field for multilingual support, But when i deleted an entry, the api still calls the entry.

So I opened the data table and found no article_id data
![Image_20220714162127](https://user-images.githubusercontent.com/5958863/179008923-02d65633-de01-4ca8-8ce7-51e7d4c8a233.png)

### To Reproduce

1. Create **article** data model with **id** and **translations** field
2. Create **title** and **body** field for **article_translations** data model
3. Click [content] -> [article] -> [Create Item]  and create an entry, save it  and delete it

Windows 10, Node v16.15.1, Directus 9.14.1, SQLite 3 database

### What version of Directus are you using?

9.14.1

### What version of Node.js are you using?

16.15.1

### What database are you using?

SQLite 3

### What browser are you using?

Chrome

### How are you deploying Directus?

localhost

**Comment by rijkvanzanten** [comment]
Can you check the relationship configuration for this field? It sounds like it's set to "nullify" instead of "delete" the related item 🙂

**Reply by kingleoric** [reply]
exactly right! thank you, [rijkvanzanten](https://github.com/rijkvanzanten)
